Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1276,9 +1276,7 @@ inline std::optional<GridTrackList> BuilderConverter::createGridTrackList(Builde
 
     auto addOne = [&](const CSSValue& currentValue) {
         if (auto* lineNamesValue = dynamicDowncast<CSSGridLineNamesValue>(currentValue)) {
-            Vector<String> names;
-            for (auto& namedGridLineValue : lineNamesValue->names())
-                names.append(namedGridLineValue);
+            Vector<String> names { lineNamesValue->names() };
```

Comment:
Can we do this without a local variable?

Reply:
Indeed it can.